Traversals/GraphBFSAdjList.java

```diff
@@ -1,3 +1,16 @@
+# Breadth First Search Using Adjacent List
```

Comment:
@Noob-Master-22  You're adding Markdown in a `.java` file. It has to be in a `.md` file.  I guess you're doing this wrong. This will break the code. 

You need to create another markdown file. And add the code over there. And then add those comments.